test/libsinsp_e2e/container_image_splitting.cpp

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
This is not e2e, please move to the unit test suite. See other comment https://github.com/falcosecurity/libs/pull/1720/files#r1509441062